src/main/java/com/internet/mixshop/model/Product.java

```diff
@@ -0,0 +1,46 @@
+package com.internet.mixshop.model;
+
+public class Product {
+    private Long id;
+    private String name;
+    private double price;
+
+    public Product(String name, double price) {
+        this.id = id;
```

Comment:
`this.id = id;` this line makes no sense

Reply:
done